debian/tests/upgrade-between-snapshots

```diff
@@ -16,7 +16,8 @@ case "$(lsb_release -s -i)" in
         # install u-u to pull in dependencies and mark python3.5-minimal
         # because otherwise u-u autoremoves it while running python3.5 code
         chroot $chroot_dir apt-get install -y unattended-upgrades python3.5-minimal
-        # TODO add a few interesting packages to upgrade
+        # add package set with many dependencies
+        chroot $chroot_dir apt-get install -y xfce4
```

Comment:
This makes the test fail exposing https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=875383